M2/Macaulay2/packages/VNumber.m2

```diff
@@ -3,7 +3,7 @@
 newPackage(
     "VNumber",
     Version => "1.0",

```

Comment:
Did you want to increase the version number as well?

Reply:
If it is not a problem, we can leave the current version. It was just a formal fix.